circular-buffer/example.erl

```diff
@@ -0,0 +1,86 @@
+-module( circular_buffer ).
```

Comment:
Same here, no spaces should be here. Doesn't matter for Erlang, but the test runner is fragile.